main.cpp

```diff
@@ -99,7 +99,8 @@ int Facility::chargeCleaningFee( int start, int laborFee )
     while( c.hoursBooked <= 10 )
     {
         ++c.hoursBooked;
-        return c.hoursBooked + laborFee;      
+        std::cout << "Add cleaning fees after " << this->hoursBooked << " hours." << std::endl;
+        return c.hoursBooked + laborFee;
     }
```

Comment:
This loop runs only once, because you return immediately the first time it runs.  
Add a condition inside the block that checks `c.hoursBooked` before returning inside the loop
- [ ] revise